sb-worker/src/main/kotlin/ru/sablebot/worker/listeners/InteractionEventsListener.kt

```
@@ -0,0 +1,83 @@
+    val coroutineLauncher: CoroutineLauncher,
+    val commandsHolderService: CommandsHolderService
+) : DiscordEventListener() {
+    companion object {
+        private val logger = KotlinLogging.logger { }
+    }
+
+    override fun onButtonInteraction(event: ButtonInteractionEvent) {
+        GlobalScope.launch {
+            val componentId = try {
+                UnleashedComponentId(event.componentId)
+            } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)